assets/upgradePatches.yaml

```diff
@@ -0,0 +1,11 @@
+---
+hcoCRPatchList:
+  - semverRange: '>=1.4.0 <=1.5.0'
+    jsonPatch: |-
+      [
+        {
+          "op": "replace",
+          "path": "/spec/featureGates/sriovLiveMigration",
+          "value": true
+        }
+      ]
```

Comment:
Since everything is json, maybe the file should be a json file as well?

Reply:
right, done